collection-scripts/gather

```diff
@@ -57,5 +57,8 @@ oc adm inspect --dest-dir must-gather "${group_resources_text}"
 # Gather Windows Kubernetes component logs
 /usr/bin/gather_windows_node_logs
 
+# Gather HAProxy config files
+/usr/bin/gather_haproxy_config
```

Comment:
Do we want this collected by default?

Reply:
Any reason not to? Do you feel that this should not be collected by default?

Reply:
I don't like collecting things in this manner (by default), but .. in this situations I don't think we have a better solution.